src/interactions/services/comments.py

```
@@ -144,12 +145,19 @@ def hide_comment(comment: Comment) -> None:
 def can_hide_comment(user: User, comment: Comment | int) -> bool:
     if isinstance(comment, int):
         comment = Comment.objects.get(id=comment)
+
+    assert isinstance(comment, Comment)
     return user == comment.author
 
 
 def can_edit_comment(user: User, comment: Comment | int) -> bool:
     if isinstance(comment, int):
-        comment = Comment.objects.get(id=comment)
+        try:
+            comment = Comment.objects.get(id=comment)
+        except Comment.DoesNotExist:
+            return False
```

We do a try/except here, but we don't in `can_hide_comment`
We should be consistent in our approach.

Personally, I think that if we are given an integer and it isn't a valid comment ID, then we should let the exception bubble up (no try/except).